nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -1038,13 +1044,15 @@ def change_attention_model(
                         att_context_size=att_context_size,
                         pos_bias_u=None,
                         pos_bias_v=None,
+                        # use_pytorch_sdpa=self.use_pytorch_sdpa,
```

Comment:
why is this commented, what is effect of sdpa for LongFormer Attention?

Reply:
I didnt add torch sdpa in longformer. I would do it in new pr i guess

Reply:
Would you please raise an error when use_pytorch_sdpa is enabled?

Reply:
done